worldedit-bukkit/src/main/java/com/fastasyncworldedit/bukkit/regions/plotsquared/PlotSquaredFeature.java

```diff
@@ -68,10 +69,20 @@ public boolean isAllowed(Player player, Plot plot, MaskType type) {
             return false;
         }
         UUID uid = player.getUniqueId();
-        return !plot.getFlag(NoWorldeditFlag.class) && (plot.isOwner(uid) || type == MaskType.MEMBER && (
-                plot.getTrusted().contains(uid) || plot.getTrusted().contains(DBFunc.EVERYONE)
-                        || (plot.getMembers().contains(uid) || plot.getMembers().contains(DBFunc.EVERYONE)) && player
-                        .hasPermission("fawe.plotsquared.member")) || player.hasPermission("fawe.plotsquared.admin"));
+        return !plot.getFlag(NoWorldeditFlag.class) && (plot.isOwner(uid) || player.hasPermission("fawe.plotsquared.admin")
+                || type == MaskType.MEMBER && (plot.getTrusted().contains(uid) || plot.getTrusted().contains(DBFunc.EVERYONE)
+                || (player.hasPermission("fawe.plotsquared.member")
+                && (plot.getMembers().contains(uid) || plot.getMembers().contains(DBFunc.EVERYONE))
+                && !plot.getOwners().isEmpty()
+                && plot.getOwners().stream().anyMatch(this::playerOnline))));
```

Comment:
Preferably we should drop a `TranslatableCaption` outlining why you can't use FAWE if one of the conditions aren't matched. Failing silently might be hard to debug until you find that specific page on the wiki listing the permissions for FAWE & P2.

Reply:
Likely to be sending two messages to players in that case